src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -159,6 +159,11 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_DELAY_STATUS {
                     type flex_delay_status;
                 }
+                leaf POLL_INTERVAL {
```

Comment:
How about adding POLL_INTERVAL leaf to every container on the same level here? Why is this fix specific to ACL?

Reply:
I added the POLL_INTERVAL leaf to ACL container, because only the ACL requires this for DPB feature.
But I will try to add this leaf to other containers and test it.

Reply:
Do you have other counters enabled in your configuration? I believe if you do, then it will also affect DPB. Don't see anything special about ACL. I think it is better to add it to this PR.

Reply:
I added POLL_INTERVAL for other containers. But I didn't find poll intervals for some container here: https://github.com/Azure/sonic-utilities/blob/master/counterpoll/main.py
For example, there is no polling interval range for RIF: https://github.com/Azure/sonic-utilities/blob/master/counterpoll/main.py#L254
Therefore, I did not add the poll interval for containers which I did not find in this file.
Maybe you know where I can find these polling intervals?

Reply:
@noaOrMlnx Please, comment on the RIF poll interval question

Reply:
@noaOrMlnx thank you.
But here only the default value of poll_interval for RIF.
Do we need a range of polling interval here for the RIF as well as for other containers? 
Or we can use only default value without range?

Reply:
Added RIF poll interval